test/distributed/fsdp/test_wrap.py

```diff
@@ -365,23 +366,28 @@ def test_always_wrap(self):
         model = FSDP(seq, process_group=self.process_group, auto_wrap_policy=always_wrap_policy)
         TestFSDPWrap.NestedSequentialModel.verify_model_all_wrapped(self, model)
 
-    @unittest.skipIf(torch.cuda.device_count() < 2, "Requires at least 2 GPUs")
```

Comment:
We should make all of these consistent and use `skip_if_lt_x_gpu()` as a follow-up.